code/procs/jobprocs.dm

```diff
@@ -680,6 +680,7 @@ var/global/totally_random_jobs = FALSE
 		src.bioHolder.mobAppearance.customization_first = new /datum/customization_style/none
 		src.bioHolder.mobAppearance.customization_second = new /datum/customization_style/none
 		src.bioHolder.mobAppearance.customization_third = new /datum/customization_style/none
+		src.update_colorful_parts()
```

Comment:
Might be enough to simply call `src.update_face()` instead?

Reply:
Doesn't seem to suffice in my testing, though i dare not speculate why at this point.

Reply:
Seems the thing that actually updates it is `H.organHolder.head.UpdateIcon()`, but I feel like maybe I shouldn't be directly calling that?

shrug, devs let me know pls